tf/deploy.tf

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "google_clouddeploy_target" "dev" {
+  location    = var.google_cloud_region
+  count       = var.use_cloud_run ? 0 : 1
```

Comment:
Can you add a comment on why we are using count? users may be confused

Reply:
Good idea. Yeah I'll add a comment. There is work to make this more intuitive in Terraform HCL, but it's still very much a WIP. See `hashicorp/terraform/issues/21953` (not linking to prevent it from being noisy in an already busy issue)